04 - Reserving/Data Simulation/_scripts/simulation.py

```diff
@@ -117,3 +122,101 @@ def neural_network_3(var1, inputs, m, textfilecontent):
     z_j[1:, :] = np.power(1 + np.exp(- W.dot(np.transpose(data2))), -1)
     pi0 = np.power(1 + np.exp(np.transpose(beta).dot(2 * z_j - 1)), -1)
     return pi0
+
+
+def cash_flow_prep_1(nop, input2, art_obs, seed1, textfilecontent):
+
+    # We only look at observations with exactly np payments
+    x_np = input2[input2['K'] == nop]
+
+    # We add an artificial observation
+    x_np.loc[len(x_np.index)] = art_obs
+
+    # We simulate the proportions paid in the positive payments
+    # Get the output of the neural network
+    x_np_nr = x_np.shape[0]
+    var1 = 'P' + nop + 'K'
+    pi_t = neural_network_2(var1, x_np, x_np_nr, textfilecontent).reshape((-1, x_np_nr), order='F')
```

Comment:
Line too long, break to go below 80 characters

Reply:
I am not commenting other lines which are too long, please see general PR comment